metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -76,7 +81,8 @@ class DebugProvider(
     stacktraceAnalyzer: StacktraceAnalyzer,
     clientConfig: ClientConfiguration,
     semanticdbs: Semanticdbs,
-    compilers: Compilers
+    compilers: Compilers,
+    supportsTestSelection: () => Boolean
```

Comment:
We will not support it in sbt?

Reply:
I want to add this to the sbt, but there's no reason to delay this for bloop.